install-manager/build.gradle.kts

```diff
@@ -2,10 +2,10 @@ plugins {
   id(GradlePluginId.ANDROID_LIBRARY)
   id(GradlePluginId.KOTLIN_ANDROID)
   id(GradlePluginId.JUNIT5_PLUGIN)
-  id(GradlePluginId.KOTLIN_KAPT)
 }
 
 android {
+  namespace = "cm.aptoide.pt.app_installer"
```

Comment:
Why this namespace? 🤔

Reply:
After updating the AGP, I had the error that namespace was not specified. so I had to move it from the manifest to build.gradle file.
From AGP 7.3, we should stop specifying the package in the manifest.

https://developer.android.com/guide/topics/manifest/manifest-element#package